### DISCUSSION TITLE
Question on quality of Training Files

### DISCUSSION THREAD

**Discussion by ChrisSmithUNR** [birdnet-team/BirdNET-Analyzer#344] [Q&A]
First, thanks so much for helping create this awesome tool!

I am a PhD student, trying to train a model to identify American pika vocalizations, and relatively new to bioacoustics.  I have a couple questions:

1. How important is it to have only high-quality training data vs sonograms with calls only faintly showing up? I have a wide range, and was wondering how it was likely to impact the model.

2.  One recorder was placed near a stream. How important is it to do a high-pass band filter to remove lower pitched background noise? (Pika frequency is well above the background)

3.  Pika vocalizations are repeated notes in various rhythms, classified as either a "short call" (a few notes) or a "long call" (usually 5+ notes strung together).  The individual notes of a "short call" and "long call" are quite similar structure...it is just the repetition and gaps in between notes that are different. Is there a way to help the model recognize the difference between these 2 calls, given that there are often few differences within the 3-second sample window?

Thanks for your help!

Image of Sonogram of calls types:
[wdfw02210.pdf](https://github.com/kahst/BirdNET-Analyzer/files/15439771/wdfw02210.pdf)

**Comment by CRodas1214** [comment]
I have the same question. I recall hearing that aplying filters to training audios was not the best idea, but I have good audios for some focal species with other singing on  background, so filtering with help a lot. Hope somebody responds this thread.
